### DISCUSSION TITLE
Bell Schedule

### DISCUSSION THREAD

**Discussion by jcarney2024** [OS4ED/openSIS-Classic#277] [Ideas]
Hello, 
I know that bell schedules are part of the ["pro" features](https://support.os4ed.com/hc/en-us/articles/202514688-Bell-Schedule-Setup) for openSIS, but as it's an essential feature for every school, I would love to see it implemented in the community version. (It also looks like it's quite outdated in the docs too.)

In the event that you don't understand what I'm talking about: this would be something like an A/B day schedule that classes follow. 

Thanks!

**Comment by openSISAdmin** [comment]
We have released a new edition of openSIS, which is a SaaS product. It has a rotating bell schedule. You can use this edition.

https://www.opensis.com

Thanks

**Reply by jcarney2024** [reply]
Hey there, 
Yes, I have seen this. The only thing holding me back from using this edition is the ability to set the time and date on the server. I frequently backdate our server to test functionality. 

Any solution to this?

**Reply by openSISAdmin** [reply]
This is a SaaS edition and is running on Azure. You cannot download and run it. However, it works on the user's browser time. So, you can backdate your computer and access the system and it will allow you to do your tests.
